docs-kits/kits/behaviour-twin-kit/changelog.md

```diff
@@ -0,0 +1,49 @@
+﻿---
+id: changelog
+title: Change Log
+description: Behaviour Twin KIT
+---
+
+<div style={{display:'block'}}>
+  <div style={{display:'inline-block', verticalAlign:'top'}}>
+
+![Behaviour Twin KIT banner](@site/static/img/kit-icons/behaviour-twin-kit-icon-mini.svg)
+
+  </div>
+  <div style={{display:'inline-block', fontSize:17, color:'rgb(255,166,1)', marginLeft:7, verticalAlign:'top', paddingTop:6}}>
+Behaviour Twin KIT
+  </div>
+</div>
+
+## [1.0.0] - 2024-05-21
+
+### ADDED
+
+#### GENERAL KIT
+
+- Initial version of the (generalized) Behaviour Twin KIT.
+- This KIT additionally contains the use cases [Remaining useful Life](use-cases/rul/overview) and [Health Indicator](use-cases/hi/overview), see section [Use Cases](use-cases/overview). The dedicated KITs of these use cases (Remaining useful Life KIT and Health Indicator KIT) are hereby deprecated.
+
+#### USE CASES
+
+- ./.
+
+### CHANGED
```

Comment:
If no content has been changed or deleted, I would leave out the bullet points.
What is the difference between KIT and Use Cases? Do we need this distinction?

Reply:
Will be done. The destinction should be preserved because use cases may be developed independently of the general part of the KIT. This provides a better overview.

Reply:
Done.